src/formatters/inline_text_formatter.js

```diff
@@ -0,0 +1,32 @@
+// Text format for easy machine parsing.
+import chalk from 'chalk';
+import columnify from 'columnify';
+
+export default function InlineTextFormatter(errorsGroupedByFile) {
+  const files = Object.keys(errorsGroupedByFile);
+
+  const errorsText = files
+    .map(file => {
+      return generateErrorsForFile(file, errorsGroupedByFile[file]);
+    })
+    .join('\n');
+  return errorsText + '\n';
+}
+
+function generateErrorsForFile(file, errors) {
+  const formattedErrors = errors.map(error => {
+    const location = error.locations[0];
+
+    return {
+      location: chalk.dim(`${file}:${location.line}:${location.column}`),
+      message: error.message,
+      rule: chalk.dim(` ${error.ruleName}`),
```

Comment:
I wonder if we should be including color in the output if it is meant to be parsed using `awk` and other tools of that nature. 🤔 My concern is that the colors might make it more difficult to parse in some cases.

What do you think?

Reply:
This is a good point, it probably should just be plain text. I'll update this.